ambry-api/src/main/java/com/github/ambry/config/ClusterMapConfig.java

```diff
@@ -154,6 +156,20 @@ public class ClusterMapConfig {
   @Config(CLUSTERMAP_CLUSTER_NAME)
   public final String clusterMapClusterName;
 
+  /**
+   * The name of the associated aggregated view cluster for this node.
+   */
+  @Config(CLUSTERMAP_AGGREGATED_VIEW_CLUSTER_NAME)
+  @Default("")
```

Comment:
You don't need to set the default.  It's not used.  Same for the default below.

Reply:
Sure. Updated